userspace/files/profile

```diff
@@ -32,32 +32,10 @@ export QT_QPA_PLATFORM="wayland-egl"
 
 # python setup
 export PYTHONPATH="/data/pythonpath"
-export PATH="/usr/local/pyenv/bin:$PATH"
-export PYENV_VERSION="3.11.4"
-export PYENV_ROOT="/usr/local/pyenv"
-
-# output of "pyenv init - --no-rehash"
-export PATH="/usr/local/pyenv/shims:${PATH}"
-export PYENV_SHELL=bash
-source '/usr/local/pyenv/libexec/../completions/pyenv.bash'
-pyenv() {
-  local command
-  command="${1:-}"
-  if [ "$#" -gt 0 ]; then
-    shift
-  fi
-
-  case "$command" in
-  activate|deactivate|rehash|shell)
-    eval "$(pyenv "sh-$command" "$@")";;
-  *)
-    command pyenv "$command" "$@";;
-  esac
-}
-
-# custom shims
-export PATH="/usr/comma/shims:${PATH}"
 
+export PATH="/usr/local/.cargo/bin:$PATH"
+export XDG_DATA_HOME="/usr/local"
+source $XDG_DATA_HOME/venv/bin/activate
```

Comment:
@adeebshihadeh This will activate the venv for all sessions on the device. It is a different behavior compared to now, where you have to use `pyenv exec`. Is this ok?

Reply:
@robin-reckmann pyenv exec would still use python "3.11.4", when run using pyenv exec
https://github.com/commaai/agnos-builder/blob/360675b9357a9ec2dbfa80d9dd5220a9ad77a4b1/userspace/files/profile#L36
unless the idea is to use different python versions across sessions, then activating venv ensures the python version 3.11.4 is consistent across sessions.